src/main/java/com/fontys/api/mutations/TeamMutation.java

```diff
@@ -19,7 +19,11 @@ public Team createTeam(String name) {
         return teamService.createTeam(name);
     }
 
-    public void deleteTeam(Long id) {
-        teamService.deleteTeam(id);
+    public String deleteTeam(Integer id) {
+        return teamService.deleteTeam(id);
+    }
+
+    public String addUserToTeam(Integer teamId, Integer userId) {
```

Comment:
this function doesn't seem to do anything

Reply:
addUserToTeam or deleteTeam?

Reply:
addUserToTeam